llvm/test/tools/sycl-post-link/virtual-functions/module-cleanup.ll

```diff
@@ -0,0 +1,41 @@
+; RUN: sycl-post-link -split=auto -properties -emit-exported-symbols \
+; RUN:     -emit-imported-symbols -emit-only-kernels-as-entry-points \
+; RUN:     -S < %s -o %t.table
+;
+; Virtual functions require some special handling during module cleanup:
+; - they are outlined into separate device image and must not be removed from
```

Comment:
```suggestion
; - they are outlined into separate device images and must not be removed from
```

Reply:
Applied in 9bef3a14ff2c8083378a19d7dd5a5c77d7ef1114